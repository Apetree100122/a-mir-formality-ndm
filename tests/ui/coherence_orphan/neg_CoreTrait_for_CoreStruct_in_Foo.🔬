[
    crate core {
        trait CoreTrait<> where [] {}
        struct CoreStruct<> where [] {}
    },
    crate foo {
        impl<> !CoreTrait<> for CoreStruct<> where [] {}
    }
]
