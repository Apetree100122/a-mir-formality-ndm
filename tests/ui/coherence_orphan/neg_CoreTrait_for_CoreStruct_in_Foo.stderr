Error: orphan_check_neg(impl ! CoreTrait for CoreStruct where [] {})

Caused by:
    failed to prove {@ IsLocal(CoreTrait(CoreStruct))} given {}, got {}
